### PR TITLE
Update summary.md

### PR DESCRIPTION
**Pull request by CyrusBelsoi**
### What does this PR do? What is the motivation?

Updates documentation to reflect new functionality

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/metrics/summary.md [hunk 1]
```diff
@@ -133,6 +133,14 @@ For any particular tag key, you can:
 
 [Learn more about tagging][5].
 
+### Metrics Related Assets
+
+ Metrics related assets refers to any dashboard, notebook, monitor or SLO that queries a particular metric. 
+1. Scroll down to the bottom of the metric’s details sidepanel, landing on the ‘Related Assets’ section.
+2. Click the dropdown button to view the type of related asset you’re interested in (i.e. dashboards, monitors, notebooks, SLOs). You can additionally leverage the search bar to quickly validate specific assets.
+   
+This information is also available via the Metrics APIs.
```
Comment (ijkaylin) [inline content/en/metrics/summary.md:142]:
```suggestion
This information is also available via API here. 
```

Make sure to link to the API directly

File: content/en/metrics/summary.md [hunk 2]
```diff
@@ -133,6 +133,14 @@ For any particular tag key, you can:
 
 [Learn more about tagging][5].
 
+### Metrics Related Assets
```
Comment (ijkaylin) [inline content/en/metrics/summary.md:136]:
```suggestion
## Metrics Related Assets
```

File: content/en/metrics/summary.md [hunk 3]
```diff
@@ -133,6 +133,14 @@ For any particular tag key, you can:
...
+
+ Metrics related assets refers to any dashboard, notebook, monitor or SLO that queries a particular metric.
```
Comment (ijkaylin) [inline content/en/metrics/summary.md:138]:
```suggestion
To determine the value of any metric name to your organization, use Metrics Related Assets. Metrics related assets refers to any dashboard, notebook, monitor or SLO that queries a particular metric. 

{{< img src="metrics/summary/IMAGEHERE.JPG" alt="Metrics Related Assets" style="width:75%;">}}
```

File: content/en/metrics/summary.md [hunk 4]
```diff
@@ -133,6 +133,14 @@ For any particular tag key, you can:
 
 [Learn more about tagging][5].
 
+### Metrics Related Assets
+
+ Metrics related assets refers to any dashboard, notebook, monitor or SLO that queries a particular metric. 
+1. Scroll down to the bottom of the metric’s details sidepanel, landing on the ‘Related Assets’ section.
```
Comment (ijkaylin) [inline content/en/metrics/summary.md:139]:
```suggestion
1. Scroll down to the bottom of the metric's details sidepanel to the "Related Assets" section
```

**Review by ijkaylin — CHANGES_REQUESTED** [review]

**Review by ijkaylin — APPROVED** [review]

**Review by aliciascott — APPROVED** [review]
Looks good, I just fixed a couple small things that the vale linter was catching, did you want this merged as well ?

**Comment by CyrusBelsoi** [comment]
> Looks good, I just fixed a couple small things that the vale linter was catching, did you want this merged as well ?

Hey @aliciascott thank you so much for the revie, yes please merge

**Pull request merged**